classes/task/sendfailednotifications_task.php

```diff
@@ -185,12 +185,24 @@ public function execute() {
 
         echo "In send failed notifications task execute".PHP_EOL;
 
+        $enablesendingnotifications = get_config('logstore_xapi', 'enablesendingnotifications');
+        if (empty($enablesendingnotifications)) {
+            echo get_string('notificationsnotenabled', 'logstore_xapi').PHP_EOL;
+            return;
+        }
+
         $results = $this->get_failed_rows();
         if (count($results) == 0) {
             echo get_string('norows', 'logstore_xapi').PHP_EOL;
             return;
         }
 
+        $notificationtrigger = get_config('logstore_xapi', 'errornotificationtrigger');
+        if (count($results) < $notificationtrigger) {
```

Comment:
should this be less than or equal to? `<=`

Reply:
I think < is correct. If $notificationtrigger is 10 it will trigger when the result set is 10 or more. Less than 10 it doesn't run.